src/cloud.c

```diff
@@ -335,6 +343,7 @@ int cloud_unregister_device(const char *id)
 	result = mq_publish_persistent_message(queue_cloud,
```

Comment:
Why do you not call `headers[0].value.value.bytes = amqp_cstring_bytes(conf->token);` before this function?

Reply:
Done